main.py

```diff
@@ -3,5 +1,31 @@
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
-def main() -> QuerySet:
-    pass
+def main() -> QuerySet[Actor]:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
```

Comment:
There is a typo in the actor's last name 'Keegan'. It should be 'Johansson'. This actor is deleted later, but it's better to avoid the typo initially.